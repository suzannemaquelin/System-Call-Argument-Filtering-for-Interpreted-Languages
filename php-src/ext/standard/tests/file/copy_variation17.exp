*** Test copy() function: With source file names containing wild-card chars ***
bool(true)
bool(true)
int(1500)

Warning: copy(%s): %s
bool(false)
bool(false)

Warning: copy(%s): %s
bool(false)
bool(false)

Warning: copy(%s): %s
bool(false)
bool(false)

Warning: copy(%s): %s
bool(false)
bool(false)
*** Done ***