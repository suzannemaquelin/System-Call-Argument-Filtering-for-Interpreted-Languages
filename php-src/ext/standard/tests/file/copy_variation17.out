Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
*** Test copy() function: With source file names containing wild-card chars ***
bool(true)
bool(true)
int(1500)

Warning: copy(/root/php-src/ext/standard/tests/file/copy*17.tmp): failed to open stream: No such file or directory in /root/php-src/ext/standard/tests/file/copy_variation17.php on line 32
bool(false)
bool(false)

Warning: copy(/root/php-src/ext/standard/tests/file/*_variation17.tmp): failed to open stream: No such file or directory in /root/php-src/ext/standard/tests/file/copy_variation17.php on line 32
bool(false)
bool(false)

Warning: copy(/root/php-src/ext/standard/tests/file/copy_variation*.tmp): failed to open stream: No such file or directory in /root/php-src/ext/standard/tests/file/copy_variation17.php on line 32
bool(false)
bool(false)

Warning: copy(/root/php-src/ext/standard/tests/file/*.tmp): failed to open stream: No such file or directory in /root/php-src/ext/standard/tests/file/copy_variation17.php on line 32
bool(false)
bool(false)
*** Done ***